src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -104,6 +104,23 @@
  * greedy will allow multiple violations in the example to be found in the same file.
  * </p>
  *
+ * <p>
+ * To configure the check to restrict an empty file:
```

Comment:
Please remove this update of doc. We do not need it, nobody request such feature to exist. It was our overengineering.

Reply:
Done.

Reply:
I so sorry, please restore doc, for some reason I though that you did it in PackageDeclaration Check doc. We do doc need it in RegexpMultilineCheck.

Reply:
No problem restored them 👍